pkg/model/bootstrapscript.go

```diff
@@ -107,7 +109,20 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup, cs *kops.Cluste
 
 			hooks := b.getRelevantHooks(cs.Hooks, ig.Spec.Role)
 			if len(hooks) > 0 {
-				spec["hooks"] = hooks
+				if hooksFingerprint, err := b.fingerprintSpecList(hooks); err != nil {
+					return "", err
+				} else {
```

Comment:
i think the else is redundant here given the return;

```shell
hooksFingerprint, err := b.fingerprintSpecList(hooks)
if err != nil {
   return "", err   
}
spec["hooksFingerprint"] = hooksFingerprint
```

Reply:
Cleaned up